impl/pkg/service/pkarr.go

```diff
@@ -106,7 +109,13 @@ func (s *PkarrService) PublishPkarr(ctx context.Context, id string, request Publ
 	if err := s.db.WriteRecord(record); err != nil {
 		return err
 	}
-	s.cache.Set(id, record, ttlcache.DefaultTTL)
+	recordBytes, err := json.Marshal(record)
```

Comment:
If you care about efficiency, do gob instead of json

Reply:
you're correct but with [goccy](https://github.com/goccy/go-json) I'm not too worried